test/extended/util/test.go

```diff
@@ -257,6 +257,10 @@ func createTestingNS(baseName string, c kclientset.Interface, labels map[string]
 
 var (
 	excludedTests = []string{
+		// these broke in the rebase, but everything else is working
+		`should function for intra-pod communication`,
```

Comment:
slightly concerning

Reply:
But we have follow-up :+1:

Reply:
> slightly concerning

Yeah, I don't think we can hold all external teams (kubelet, storage, cluster-lifecycle, scheduler, controllers, builds, etc) on these four related tests.  We narrowed it down to a scheduling failure.